### PR TITLE
tifffile: Build from scratch.

### PR DESCRIPTION
**Pull request by jakirkham**
Tested on Mac OS 10.9 and CentOS 6.3. Unable to test on Windows, but expect it should work.

### CODE REVIEW

File: tifffile/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    #
+    # Would create an entry point called tifffile that calls tifffile.main()
+
+
+  # If this is a new build for the same version, increment the build
+  # number. If you do not include this key, it defaults to 0.
+  # number: 1
+
+requirements:
+  build:
```
Comment (stuarteberg) [inline tifffile/meta.yaml]: These requirements aren't right:

- IIUC, `libtiff` is not a dependency of `tifffile` (not even an optional dependency).
- The `tifffile.py` docstring is specific about version requirements:
  - `python 2.7*|>=3.4`
  - `numpy >=1.8.2`
- `matplotlib` is an optional dependency.  I think it should not be listed here. User who need plotting can install it separately.

**Comment by stuarteberg** [comment]
This PR is a dupe of #195, which also lacks proper version requirements.

**Comment by jakirkham** [comment]
Actually, I am going to close this as this has been included in `skimage`. See code here ( https://github.com/scikit-image/scikit-image/tree/master/skimage/external/tifffile ) and API ( http://scikit-image.org/docs/dev/api/skimage.external.tifffile.html ). Note they simply include tifffile directly so it is not even a dependency.

**Comment by jakirkham** [comment]
@stuarteberg I don't follow your complaint regarding #195. It appears that they have included `numpy` and `python` without anything additional. Is it simply that you are wishing to see version strings? Regardless, I think this is moot point.

**Comment by jakirkham** [comment]
Sorry of the dupe.

**Comment by stuarteberg** [comment]
>this has been included in skimage.

Excellent.  Good to know, thanks.

Regarding #195:

>Is it simply that you are wishing to see version strings?

Yes.

**Comment by jakirkham** [comment]
To be honest, I think I ran `conda skeleton tifffile` and just committed it.

**Pull request closed without merging**